R/pkg/R/DataFrame.R

```diff
@@ -41,7 +41,7 @@ setOldClass("structType")
 #'\dontrun{
 #' sparkR.session()
 #' df <- createDataFrame(faithful)
-#'}
+#' }
```

Comment:
Yeah lets not do them in this PR. If we want to fix this lets do them in a separate PR ?